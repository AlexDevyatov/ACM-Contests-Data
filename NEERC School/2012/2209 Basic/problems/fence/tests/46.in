14
-344 -245
-238 -326
-208 -337
-189 -340
-44 -342
248 -324
303 -173
313 -133
338 136
314 262
255 316
20 342
-204 339
-342 219
201 -170
