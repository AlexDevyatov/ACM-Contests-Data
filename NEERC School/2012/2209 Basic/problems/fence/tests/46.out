15
-344 -245
-238 -326
-208 -337
-189 -340
-44 -342
248 -324
303 -173
201 -170
313 -133
338 136
314 262
255 316
20 342
-204 339
-342 219
